keyboards/splitkb/aurora/corne/corne.c

```diff
@@ -85,7 +85,7 @@ static void render_mod_status_gui_alt(uint8_t modifiers) {
         oled_write_P(gui_off_2, false);
     }
 
-    if (modifiers & MOD_MASK_GUI & MOD_MASK_ALT) {
+    if ((modifiers & MOD_MASK_GUI) && (modifiers & MOD_MASK_ALT)) {
```

Comment:
Note that the same condition is actually written correctly just above, probably this one was forgotten as it's easy to miss.